docker_images_metadata.json

```diff
@@ -0,0 +1,1486 @@
+         },
+         "1.0.0.26289":{
+            "python_version":"3.9.6"
+         }
+      },
+      "sklearn":{
+         "1.0.0.64885":{
+            "python_version":"3.10.8"
+         },
+         "1.0.0.12410":{
+            "python_version":"3.8.6"
+         },
+         "1.0.0.29944":{
+            "python_version":"3.8.6"
+         },
+         "1.0.0.49796":{
+            "python_version":"3.10.8"
+         }
+      },
+      "pcap-miner":{
+         "1.0.0.32154":{
+            "python_version":"3.9.6"
+         },
+         "1.0.0.10664":{
+            "python_version":"3.8.5"
+         },
+         "1.0.0.9769":{
+            "python_version":"3.8.3"
+         },
+         "1.0.0.30520":{
+            "python_version":"3.9.6"
+         }
+      },
+      "readpdf":{
+         "1.0.0.43274":{
+            "python_version":"3.10.9"
+         },
+         "1.0.0.50963":{
+            "python_version":"3.10.10"
+         }
+      },
+      "iputils":{
+         "1.0.0.4663":{
+            "python_version":"3.7.5"
+         }
+      },
+      "unrar":{
+         "1.4":{
+            "python_version":"2.7.12"
+         }
+      },
+      "processing-image-file":{
+         "1.0.0.66515":{
+            "python_version":"3.10.12"
+         }
+      },
+      "office-utils":{
+         "2.0.0.54910":{
+            "python_version":"3.10.11"
+         },
+         "2.0.0.49835":{
+            "python_version":"3.10.10"
+         }
+      },
+      "aquatone":{
+         "2.0.0.36846":{
+            "python_version":"3.10.7"
+         }
+      },
+      "pcap-http-extractor":{
+         "1.0.0.32113":{
+            "python_version":"3.9.6"
+         }
+      },
+      "nltk":{
+         "2.0.0.19143":{
+            "python_version":"3.9.4"
+         }
+      },
+      "dempcap":{
+         "1.0.0.14059":{
+            "python_version":"2.7.18"
+         }
+      },
+      "docxpy":{
+         "1.0.0.40261":{
+            "python_version":"3.10.8"
+         },
+         "1.0.0.33689":{
+            "python_version":"3.10.6"
+         }
+      },
+      "ssdeep":{
+         "1.0.0.23743":{
+            "python_version":"3.9.6"
+         }
+      },
+      "machine-learning":{
+         "1.0.0.22015":{
+            "python_version":"2.7.18"
+         }
+      },
+      "xslxwriter":{
+         "1.0.0.45070":{
+            "python_version":"3.10.9"
+         }
+      },
+      "mlclustering":{
+         "1.0.0.23151":{
+            "python_version":"3.9.6"
+         }
+      },
+      "sane-pdf-reports":{
+         "1.0.0.62999":{
+            "python_version":"3.10.12"
+         }
+      },
+      "fetch-data":{
+         "1.0.0.22177":{
+            "python_version":"3.9.5"
+         }
+      },
+      "rakyll-hey":{
+         "1.0.0.49364":{
+            "python_version":"3.10.10"
+         }
+      },
+      "archer":{
+         "1.0.0.270":{
+            "python_version":"3.7.3"
+         }
+      },
+      "powershell-teams":{
```

Comment:
why it's empty here? if it doesn't have any tags, just don't keep it

Reply:
removed them